os/drivers/kernel_test/kernel_test_codes/Make.defs

```diff
@@ -17,9 +17,9 @@
 ############################################################################
 # Include kernel test drivers
 
+CSRCS += kernel_test_drv.c
```

Comment:
This should be at kernel_test folder, not in kernel_test_codes. This is not a kernel testcase. This is a driver.

Reply:
done